### ISSUE TITLE
Alt+Tab, the layout is correct?

### ISSUE THREAD

**Issue by diegomartarelo** [vinceliuice/Orchis-theme#292]
Good night, king of my most loved themes! When I do an alt+tab, it doesn't look very nice, when the window sentence is too long, it stays out of the layout, make the border smaller, make the border softer, not so aggressive... please, if you can take a look at this question for me! Thank you in advance for developing and maintaining this wonderful theme!

Forgive me, it could be some kind of system update that resulted in this, I'm using ubuntu 23.04 with the gnome 43.2 version and the configuration defined for updates in development (beta) .. Could there have been any modifications by the system?

**Comment by vinceliuice** [comment]
I’ll check this

**Comment by CJTS15** [comment]
Hi @diegomartarelo @vinceliuice , I found out how to adjust the radius of the window switcher or the container box when you do alt tab. At first the border radius of the container is very harsh, I had to take a look at the theme files and stumble upon gnome-shell.css. Search for workspace-switcher-container, and edit the border-radius, save file. To see the changes, change to another theme and then go back to Orchis, you'll see the border radius change when you alt tab.
